tests/Load/package.json

```diff
@@ -1,27 +1,27 @@
 {
   "name": "Load",
   "version": "1.0.0",
-  "description": "",
+  "description": "Load testing for VilnaCRM user-service",
   "main": "index.js",
+  "type": "module",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "prepare-users": "cd ../../.. && docker run -v ./tests/Load:/loadTests --net=host --rm --user $(id -u) k6 run /loadTests/utils/prepareUsers.js",
```

Comment:
Why do we need this new commands ?

Reply:
You created the commands, but I don’t see where are you using these commands
So I don’t think that we need these commands at all there 
Please remove them